CONTRIBUTING.md

```diff
@@ -4,3 +4,7 @@ Before submitting a pull request, please verify that:
 * Any strings are extracted for [translation](http://projects.theforeman.org/projects/foreman/wiki/Translating).
 
 For more information regarding contributions, plese read the [Foreman Handbook](http://theforeman.org/handbook.html) and [Foreman contributions](http://theforeman.org/contribute.html).
+
+Looking for a good starting issue to work on? have a look at: [Easy Fix Issues](https://projects.theforeman.org/projects/foreman/issues?utf8=%E2%9C%93&set_filter=1&sort=id%3Adesc&f%5B%5D=status_id&op%5Bstatus_id%5D=o&f%5B%5D=cf_3&op%5Bcf_3%5D=%3D&v%5Bcf_3%5D%5B%5D=trivial&v%5Bcf_3%5D%5B%5D=easy&f%5B%5D=&c%5B%5D=tracker&c%5B%5D=status&c%5B%5D=priority&c%5B%5D=subject&c%5B%5D=author&c%5B%5D=assigned_to&c%5B%5D=updated_on&c%5B%5D=category&c%5B%5D=fixed_version&group_by=)
```

Comment:
Maybe we also want to filter the status and display only `NEW` issues?

Reply:
For instance from this year? half a year? month? 
I think we should present those from 2022, because they would be the most relevant ones.
Like this: [Added another filter](https://projects.theforeman.org/projects/foreman/issues?utf8=%E2%9C%93&set_filter=1&sort=id%3Adesc&f%5B%5D=status_id&op%5Bstatus_id%5D=o&f%5B%5D=cf_3&op%5Bcf_3%5D=%3D&v%5Bcf_3%5D%5B%5D=trivial&v%5Bcf_3%5D%5B%5D=easy&f%5B%5D=updated_on&op%5Bupdated_on%5D=%3E%3D&v%5Bupdated_on%5D%5B%5D=2022-01-01&f%5B%5D=&c%5B%5D=tracker&c%5B%5D=status&c%5B%5D=priority&c%5B%5D=subject&c%5B%5D=author&c%5B%5D=assigned_to&c%5B%5D=updated_on&c%5B%5D=category&c%5B%5D=fixed_version&group_by=)
What do you think @nofaralfasi ?

Reply:
Can you add a filter for `Assignee : none`? as we don't want 2 people to work on the same task

Reply:
Sure those are good ideas thanks! 
I'll add those filters
Like this: [Added more filters](https://projects.theforeman.org/projects/foreman/issues?utf8=%E2%9C%93&set_filter=1&sort=id%3Adesc&f%5B%5D=status_id&op%5Bstatus_id%5D=%3D&v%5Bstatus_id%5D%5B%5D=1&f%5B%5D=cf_3&op%5Bcf_3%5D=%3D&v%5Bcf_3%5D%5B%5D=trivial&v%5Bcf_3%5D%5B%5D=easy&f%5B%5D=assigned_to_id&op%5Bassigned_to_id%5D=%21*&f%5B%5D=&c%5B%5D=tracker&c%5B%5D=status&c%5B%5D=priority&c%5B%5D=subject&c%5B%5D=author&c%5B%5D=assigned_to&c%5B%5D=updated_on&c%5B%5D=category&c%5B%5D=fixed_version&group_by=)